cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -529,23 +535,15 @@ private StepModellingResult compileStep(
 
         replaceOnFailureReference(postStepData, onFailureStepName);
 
-        @SuppressWarnings("unchecked")
-        List<Argument> arguments = (List<Argument>) preStepData.get(SlangTextualKeys.DO_KEY);
 
         String refId = "";
-        Map<String, Object> doRawData;
-        try {
-            //noinspection unchecked
-            doRawData = (Map<String, Object>) stepRawData.get(SlangTextualKeys.DO_KEY);
-        } catch (ClassCastException ex) {
-            doRawData = new HashMap<>();
-        }
-        if (MapUtils.isEmpty(doRawData)) {
-            errors.add(new RuntimeException("Step: \'" + stepName + "\' has no reference information"));
```

Comment:
why did you remove this exception from the errors list?

Reply:
Because we added it at the transformers level (DoTransformer line 25)